prawcore/requestor.py

```diff
@@ -19,6 +19,7 @@ def __init__(
         user_agent,
         oauth_url="https://oauth.reddit.com",
         reddit_url="https://www.reddit.com",
+        timeout=TIMEOUT,
```

Comment:
Will you add this after `session` please? That happens to keep them sorted, and also means it's not a breaking change.

Reply:
I've changed it